src/models/operationDetails.ts

```diff
@@ -103,6 +103,7 @@ export interface OperationSpecDetails {
 export interface OperationGroupDetails {
   key: string;
   name: string;
+  originalKey: string;
```

Comment:
what is the purpose of this `originalKey`? This name does not convey the true meaning and might create confusion in the future.  Could you explain?

Reply:
This is because you have changed the name in conflict resolver and I need the originalKey value to find out which operationGroup/operation matches to one specific example in test modeler.